lib/chat_web/channels/room_channel.ex

```diff
@@ -22,26 +22,30 @@ defmodule ChatWeb.RoomChannel do
   # broadcast to everyone in the current topic (room:lobby).
   @impl true
   def handle_in("shout", payload, socket) do
-    Chat.Message.changeset(%Chat.Message{}, payload) |> Chat.Repo.insert
+    Chat.Message.changeset(%Chat.Message{}, payload) |> Chat.Repo.insert()
     broadcast(socket, "shout", payload)
     {:noreply, socket}
   end
 
-  # Add authorization logic here as required.
-  # Auth coming soon via: https://github.com/dwyl/phoenix-chat-example/issues/54
-  # defp authorized?(_payload) do
-  #   true
-  # end
-
   @impl true
   def handle_info(:after_join, socket) do
     Chat.Message.get_messages()
-    |> Enum.reverse() # reverse to display the latest message at the bottom of the page
-    |> Enum.each(fn msg -> push(socket, "shout", %{
+    # revers to display the latest message at the bottom of the page
+    |> Enum.reverse()
+    |> Enum.each(fn msg ->
+      push(socket, "shout", %{
         name: msg.name,
         message: msg.message,
         inserted_at: msg.inserted_at
-      }) end)
-    {:noreply, socket} # :noreply
+      })
+    end)
+
+    # :noreply
+    {:noreply, socket}
+  end
+
+  # Add authorization logic here as required.
+  defp authorized?(_payload) do
+    true
   end
```

Comment:
Is this function just a placeholder for future code logic? Or just an example how we can add authorization code?

Reply:
This is boilerplate code generated by Phoenix when running `mix phx.gen.channel Room`. I didn't change it as it wasn't relevant to the tutorial 👍